CONTRIBUTING.md

```diff
@@ -424,72 +440,84 @@ KUBERNETES_VERSION==v1.13.0 BACKEND=postgres ENV=kubernetes ./scripts/ci/local_c
 
 * PYTHON_VERSION might be one of 3.5/3.6
 * BACKEND might be one of postgres/sqlite/mysql
-* ENV might be one of docker/kubernetes
+* ENV might be one of docker/kubernetes/bare
 * KUBERNETES_VERSION - required for Kubernetes tests - currently KUBERNETES_VERSION=v1.13.0.
 
-The kubernetes env might not work locally as easily as other tests because it requires your host
-to be setup properly (specifically it installs minikube cluster locally on your host and depending 
+The following environments are possible:
+ * The "docker" environment (default) - it starts all dependencies required by full suite of tests
```

Comment:
* Docker (default): starts all dependencies required by full integration test-suite (postgres, mysql, celery, etc.). This option is resource intensive so do not forget to [stop the environment](#stopping-the-environment) when you are finished. This option is also RAM intensive and can slow down your machine.